client/post-editor/term-token-field/index.jsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import cloneDeep from 'lodash/cloneDeep';
+import unescapeString from 'lodash/unescape';
```

Comment:
in the `<TermTreeSelector />` and in other places we are using [https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/term-tree-selector/terms.jsx#L21](decodeEntities) - not sure if that would work here as well or not.